editor/src/components/context-menu-items.ts

```diff
@@ -135,9 +135,10 @@ export const toggleShadowItem: ContextMenuItem<CanvasData> = {
 }
 
 export const setAsFocusedElement: ContextMenuItem<CanvasData> = {
-  name: 'Edit Component',
-  // todo only enable if thing is component
-  enabled: true,
+  name: 'Set As Focused Element',
+  enabled: (data) => {
+    return true
```

Comment:
I think this is still not going to be good enough here :)

Reply:
that's for the next PR! :D I should have split it into 2 branches :(